operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -24,7 +24,8 @@ spec:
 #  additional_display_details: []
 #
 # A list of additional details that Kiali will look for in annotations and display, for every workloads and services, in their respective details pages.
-# Each item in the list is an object with "annotation" and "title" fields to indicate which annotation is to be displayed and how it should be displayed.*
+# Each item in the list is an object with "annotation", "title" and "icon_annotation" fields to indicate which annotation is to be displayed and how it should be displayed.
+# At the moment, "icon_annotation" possible values are "rest", "grpc" and "graphql".
```

Comment:
This is trivial, but I just noticed that this example below doesn't show "icon_annotation" - is this a required field? If so, should probably add it to the example below.

Reply:
It's an optional field, but I've improved the documentation anyway